apps/studio/components/interfaces/HomeNew/GettingStarted/GettingStarted.utils.tsx

```diff
@@ -0,0 +1,401 @@
+          onClick: () =>
+            openAiChat(
+              'Generate RLS policies',
+              'Generate RLS policies for my existing tables in the public schema and guide me through the process of adding them as migration files to my codebase '
+            ),
+        },
+      ],
+    },
+    {
+      key: 'setup-auth',
+      status: hasConfiguredAuth ? 'complete' : 'incomplete',
+      title: 'Configure authentication',
+      icon: <User strokeWidth={1} className="text-foreground-muted" size={16} />,
+      description:
+        "It's time to configure your authentication providers and settings for Supabase Auth, so jump into the configuration page and tailor the providers you need.",
+      actions: [{ label: 'Configure', href: `/project/${ref}/auth/providers`, variant: 'default' }],
+    },
+    {
+      key: 'connect-app',
+      status: hasAppConnected ? 'complete' : 'incomplete',
+      title: 'Connect your application',
+      icon: <Code strokeWidth={1} className="text-foreground-muted" size={16} />,
+      description:
+        'Your project is ready; use the framework selector to preview starter code and launch the Connect flow with the client library you prefer.',
+      actions: connectActions,
+    },
+    {
+      key: 'signup-first-user',
+      status: hasFirstUser ? 'complete' : 'incomplete',
+      title: 'Sign up your first user',
+      icon: <UserPlus strokeWidth={1} className="text-foreground-muted" size={16} />,
+      description:
+        'Test your authentication setup by creating the first user account, following the docs if you need a step-by-step walkthrough.',
+      actions: [
+        {
+          label: 'Read docs',
+          href: 'https://supabase.com/docs/guides/auth',
+          variant: 'default',
+        },
+      ],
+    },
+    {
+      key: 'upload-file',
+      status: hasStorageObjects ? 'complete' : 'incomplete',
+      title: 'Upload a file',
+      icon: <Upload strokeWidth={1} className="text-foreground-muted" size={16} />,
+      description:
+        'Integrate file storage by creating a bucket and uploading a file, starting from the buckets dashboard linked below.',
+      actions: [{ label: 'Buckets', href: `/project/${ref}/storage/buckets`, variant: 'default' }],
+    },
+    {
+      key: 'create-edge-function',
+      status: hasEdgeFunctions ? 'complete' : 'incomplete',
+      title: 'Deploy an Edge Function',
+      icon: <Code strokeWidth={1} className="text-foreground-muted" size={16} />,
+      description:
+        'Add server-side logic by creating and deploying your first Edge Function—a lightweight TypeScript or JavaScript function that runs close to your users—then revisit the list to monitor and iterate on it.',
+      actions: [
+        {
+          label: 'Create a function',
+          href: `/project/${ref}/functions/new`,
+          variant: 'default',
+        },
+        { label: 'View functions', href: `/project/${ref}/functions`, variant: 'default' },
```

Comment:
likewise here - given that this is part of the code workflow, should the CTA here be to use the CLI? or the dashboard